c++/src/Vector.cc

```diff
@@ -242,7 +242,6 @@ namespace orc {
     }
     return false;
   }
-
```

Comment:
Could you revert this irrelevant change, @rizaon ?

Reply:
Reverted.